en/python_installation/instructions.md

```diff
@@ -50,7 +50,7 @@ $ python3 --version
 Python 3.6.1
 ```
 
-If you have a different 'micro version' of Python installed, e.g. 3.6.0, then you don't have to upgrade. If you don't have Python installed, or if you want a different version, you can install it as follows:
+If you have a different version of Python installed, at least 3.4.x (e.g. 3.6.0), then you don't have to upgrade. If you don't have Python installed, or if you want a newer version, you can install it as follows:
```

Comment:
I have doubts about the 3.4.x. It implies to me that you can only change the x. How about 3.4.0?

Reply:
Good thought, fixed that in next commit.